source/bluetooth/MicroBitBLEManager.cpp

```diff
@@ -267,6 +277,9 @@ void MicroBitBLEManager::init(ManagedString deviceName, ManagedString serialNumb
     // automatically restart advertising after a device disconnects.
     ble->gap().onDisconnection(bleDisconnectionCallback);
     ble->gattServer().onSysAttrMissing(bleSysAttrMissingCallback);
+    
+    // generate an event when a Bluetooth connection is lost
```

Comment:
I think this comment should read:

"generate an event when a Bluetooth device connects" or something along those lines.